### DISCUSSION TITLE
Support for maven?

### DISCUSSION THREAD

**Discussion by so87** [DataDog/guarddog#284] [General]
I am wondering if there is support for maven. If not, are there any other open source(or paid) tools that do this?

**Comment by christophetd** [comment]
Hey there! No immediate plans from our side. Are you aware of malicious Maven packages in the wild? Would be curious to hear about the prevalence

**Reply by so87** [reply]
No i haven't heard of any malicious packages but i also haven't looked. If there are malicious npm packages, why wouldn't there be malicious maven packages? Is their vetting process such much better that there is no risk?

**Reply by AngellusMortis** [reply]
Maven is not as centralized as NPM or PyPi. There are public maven repos, but there is never any guarantee that anyone will use a specific one. Also, the process to _get access_ to most maven repos are a lot more involved then NPM or PyPi. I can make an account on PyPi and upload a new package 100% self-service and relatively quickly. Maven Central, for example, [requires manual approval](https://central.sonatype.org/publish/#correct-coordinates) for users to upload packages.
